django_enum_choices/fields.py

```diff
@@ -73,7 +73,16 @@ def build_choices(self) -> Tuple[Tuple[str]]:
         )
 
     def _calculate_max_length(self, **kwargs) -> int:
-        max_choice_length = max(len(choice) for choice, _ in kwargs['choices'])
+        # Checking whether the provided choice builder is an django_enum_choices.choice_builder.attribute_value
+        # since it will have the enum's field name as a first value, not the value of the field itself
+        has_attribute_value_choice_builder = self._original_choice_builder.__name__ == 'attribute_value'
```

Comment:
Using builder `__name__` looks extremely fragile to me. I wonder if it's possible to look inside the actual choices.

Reply:
You're absolutely right, thanks for the suggestion.